docs/components/Breadcrumbs.tsx

```diff
@@ -76,9 +76,10 @@ export function Breadcrumbs() {
             <Link href={href}>
               <a>
                 <Type
-                  look="body12"
+                  look="body14"
                   css={{
-                    textTransform: 'uppercase',
+                    textTransform: 'capitalize',
```

Comment:
I think we might need to be a bit more specific about how we generate our breadcrumbs. I don't think `capitalize` is quite what we want. Some of our titles have specific casing requirements, so we might be better off passing through explicit values in these cases

![image](https://user-images.githubusercontent.com/616382/123879165-d60c3700-d983-11eb-99a6-8c70c0a001b4.png)
![image](https://user-images.githubusercontent.com/616382/123879176-dc021800-d983-11eb-9ba6-f7f9b58c1837.png)

Reply:
I think our best bet here is to leave this as `uppercase` for now and revisit this in a separate PR.

Reply:
ok damn didnt think about that! I was thinking about just leaving it as written in the markdown, but its not consistent there either... `uppercase` it is then